docs/application/native/guides/app-management/sticker.md

```diff
@@ -0,0 +1,431 @@
+
+### Updating the sticker information
+
+You can update the sticker information stored in the Sticker database using the following functions:
+
+```
+void
+update_sticker_data(sticker_provider_h sticker_provider, sticker_data_h sticker_data)
+{
+    int ret;
+    ret = sticker_provider_update_data(sticker_provider, sticker_data);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+}
+```
+
+### Destroying a Sticker provider handle
+
+Destroy the Sticker provider handle using the `sticker_provider_destroy()` function:
+
+```
+void
+destroy_sticker_provider_handle(sticker_provider_h sticker_provider)
+{
+    int ret;
+    ret = sticker_provider_destroy(sticker_provider);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+}
+```
+
+<a name="consumer_application"></a>
+## Creating Sticker consumer application
+
+You can create the Sticker consumer application using Sticker API to use sticker information. You can search for stickers and get details of stickers.
+
+### Prerequisites
+
+To enable your Sticker consumer application to use the Sticker functionality:
+
+1. To use the Sticker consumer API, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/mediastorage</privilege>
+    </privileges>
+    ```
+
+2. To use the functions and data types of the Sticker API, include the `<sticker_consumer.h>` header file in your application:
+
+    ```
+    #include <sticker_consumer.h>
+    ```
+
+ To ensure that a Sticker function has been executed properly, make sure that the return is equal to `STICKER_ERROR_NONE`.
+
+### Creating a Sticker consumer handle
+
+Create the Sticker consumer handle using the `sticker_consumer_create()` function that allows you to retrieve the sticker information:
+
+```
+void
+create_sticker_consumer_handle()
+{
+    sticker_consumer_h sticker_consumer;
+    int ret;
+    ret = sticker_consumer_create(&sticker_consumer);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+}
+```
+
+<a name="retrieve_sticker"></a>
+### Retrieving the sticker information
+
+You can retrieve for all stickers stored in Sticker database and use keyword, group name, and type of URI to retrieve stickers.
+
+The following examples show how to retrieve stickers using group name:
+
+```
+void
+sticker_info_cb(sticker_data_h sticker_data, void *user_data)
+{
+    /* Your code */
+}
+
+void
+retrieve_stickers(sticker_consumer_h sticker_consumer)
+{
+    int ret;
+    int offset;
+    int count;
+    int result;
+    char *group_name;
+
+    /* Your code */
+
+    /* Retrieve all sticker data by group name */
+    ret = sticker_consumer_data_foreach_by_group(sticker_consumer, offset, count, &result, group_name, sticker_info_cb, NULL);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+}
+```
+
+You can retrieve for keywords and group name of the stickers.
```

Comment:
You can retrieve the keywords and the group name of the stickers.

Reply:
Fixed.